### ISSUE TITLE
Cannot download from Youtube

### ISSUE THREAD

**Issue by Bakuthecat** [imputnet/cobalt#759]
### bug description

Hello. For some reason I'm unable to download any video from YouTube. Any time I try to download one, it gives me an error saying "i can't access this video. it may be restricted on youtube's side. have you pasted the right link?"

I have been pasting the right links as told, but it still gives me that error. What's causing this?

### reproduction steps

1. Go to youtube.com
2. Copy any link from the site
3. Go to cobalt.tools
4. Paste link from Youtube
5. See error that says "i can't access this video. it may be restricted on youtube's side. have you pasted the right link?"

### screenshots

The error whenever I try to paste a link to a Youtube video.
![Capture](https://github.com/user-attachments/assets/8d77470c-29e3-447a-8303-156d30b69283)

### platform information

Windows 10 Home, Microsoft Edge Version 128

**Comment by cg505** [comment]
been seeing this today and maybe yesterday?

from da network tab:

POST to https://api.cobalt.tools/session (no payload) gives 400 with
```
{"status":"error","error":{"code":"error.api.auth.not_configured"}}
```
(maybe this is expected?)

POST to https://api.cobalt.tools/ (e.g. `{"url":"blahblah","videoQuality":"720","alwaysProxy":true or absent}`) gives 400 with
```
{"status":"error","error":{"code":"error.api.content.video.unavailable","context":{"service":"youtube"}}}
```

verified that the video I'm testing with (e.g. https://www.youtube.com/watch?v=uXhjLUPyxw8) is available

it does sporadically work, idk what the difference is

**Comment by lostdusty** [comment]
`/session` endpoint has nothing to do with this issue. This is happening because the cookies used by cobalt expired, youtube likely changed something about what its returning, and cobalt is wrongly saying that the video is privated (or other similar issue), instead of saying the cookies expired

**Comment by wukko** [comment]
oops

**Comment by BentleyBentayga** [comment]
![image](https://github.com/user-attachments/assets/cb0da2e3-6502-4430-b29c-0786e90cf53c)
having the same problem WHEN IM TRYING TO DOWNLOAD MICKEY 17 😭😭😭

**Comment by wukko** [comment]
please be patient im working on it

**Comment by wukko** [comment]
i've refreshed dummy accounts for youtube and the issue is now resolved

**Comment by Acceleration2299** [comment]
Uhhh, Apologies for the Necro? idk how this works.

Cobalt's new processing queue seems to be "fucked", for lack of better explanation. I tried to download a song, didn't work, tried to download a short, didn't work. Tried a music *video*, Also didnt work, All of these have returned- shit i lost them.

I tried again, and now the site only returns this.
![Image](https://github.com/user-attachments/assets/25152a8b-f4a5-448d-aa14-bc93c991e022)

I will note, that i'm using multiple plugins on firefox, namely uBlock Origin, SponsorBlock, Return YT Dislike, and Privacy Badger. If this is relevant.
